oms/src/main/java/com/gfg/oms/response/GeneralMessage.java

```diff
@@ -0,0 +1,9 @@
+package com.gfg.oms.response;
+
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@NoArgsConstructor
+@Data
+abstract public class GeneralMessage {
```

Comment:
Is this needed if there are no abstract methods?

Reply:
I want to force all responses to be extended from `Response`. Also, `message` is common across all OMS responses and I want every child of `Response` to have `message`. Unfortunately, there is zero in common between `message` structure of every response, therefore GeneralMessage has no properties.